hack/lib/build/constants.sh

```diff
@@ -53,6 +53,9 @@ readonly OPENSHIFT_BINARY_SYMLINKS=(
   openshift-recycle
   openshift-sti-build
   openshift-docker-build
+  openshift-git-clone
+  openshift-manage-dockerfile
+  openshift-extract-image-content
```

Comment:
i kinda dislike the api name since this feature is frequently used for things that i would not consider source (binary libraries, jars, etc) so i was trying to pick something that seemed more suitable to what it's for, but if you can get a second vote i'll change it.